.github/workflows/bazel.yml

```diff
@@ -6,7 +6,7 @@ name: bazel
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
   push:
-    branches: [main]
+    branches: [master]
```

Comment:
BTW, I want to change the base branch from master to main

Reply:
Sounds awesome!